### PR TITLE
Elasticsearch Transcript Store component

### PR DESCRIPTION
**Pull request by pratikpanda**
Elasticsearch based transcript store component for bots created using Microsoft Bot Framework.

### CODE REVIEW

**Comment by pratikpanda** [comment]
@garypretty can you look into this one?

File: libraries/Bot.Builder.Community.TranscriptStore.Elasticsearch/README.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
+﻿# Bot.Builder.Community.TranscriptStore.Elasticsearch
```
Comment (garypretty) [inline libraries/Bot.Builder.Community.TranscriptStore.Elasticsearch/README.md]: Can you get the readme into the same format as the other storage provider (which I updated to be consistent with the rest of the repo). Thanks.

Comment (pratikpanda) [reply libraries/Bot.Builder.Community.TranscriptStore.Elasticsearch/README.md]: Sure, I will update the readme.

**Comment by pratikpanda** [comment]
@garypretty, I have updated the README and fixed the project type for the test projects as well. Just a note that these are integration tests and therefore are dependent on Elasticsearch server. I am skipping these tests as part of the build process.

**Pull request closed without merging**